Build-LibLlvm.ps1

```
@@ -0,0 +1,59 @@
+Param(
+    [string]$Configuration="Release"
+)
+
+pushd $PSScriptRoot
+$oldPath = $env:Path
+try
+{
+    . ./buildutils.ps1
+    $buildInfo = Initialize-BuildEnvironment
+
+    # Need to invoke NuGet directly for restore of vcxproj as /t:Restore target doesn't support packages.config
+    # and PackageReference isn't supported for native projects... [Sigh...]
+    Write-Information "Restoring LibLLVM"
+    Invoke-NuGet restore (Join-Path src Interop LibLLVM LibLLVM.vcxproj)
+
+    $buildOutputDir = Join-Path src Interop LibLLVM BuildOutput
+    if (Test-Path $buildOutputDir) {
+        Write-Verbose "Cleaning out the old data from $buildOutputDir"
+        Remove-Item -Path $buildOutputDir -Recurse -Force | Out-Null
+    }
+    New-Item -Path $buildOutputDir -ItemType Directory
+
+    $target = "all"
+    if ($buildInfo['Platform'] -eq [platform]::Windows) {
+        $target = "ALL_BUILD"
+    }
+    Write-Information "Building LibLLVM"
+    cd $buildOutputDir
+    cmake (Resolve-Path ..)
+    if($LASTEXITCODE -ne 0 )
+    {
+        throw "LibLLVM CMake generate exited with code: $LASTEXITCODE"
+    }
+    cmake --build . --target $target --config $Configuration
+    if($LASTEXITCODE -ne 0 )
+    {
+        throw "LibLLVM CMake build exited with code: $LASTEXITCODE"
+    }
+    cd $PSScriptRoot
+
+    if ($buildInfo['Platform'] -eq [platform]::Windows) {
+        New-Item -ErrorAction SilentlyContinue -ItemType Directory -Path (Join-Path $buildInfo["NativeXplat"] win-x64)
```

use md please, also do not set Error action to SilentlyContinue as it hides any failures to create the directory.